model-generator/plugin/src/main/java/com/vimeo/modelgenerator/tasks/GenerateModelsTask.kt

```diff
@@ -48,12 +48,15 @@ open class GenerateModelsTask : DefaultTask() {
     private val output =
         project.convention.getPlugin(JavaPluginConvention::class.java).sourceSets.maybeCreate("main").java.srcDirs.first().path
```

Comment:
```suggestion
        project.extensions.getByType(JavaPluginExtension::class.java).sourceSets.maybeCreate("main").java.srcDirs.first().path
```

**Optional:** This was in my PR (#516), the `convention` getter and `JavaPluginConvention` have both been deprecated so this is the recommended replacement.